lib/routes/utils/oauth.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const encrypt = require('../../../fxa-oauth-server/lib/encrypt');
```

Comment:
Is this going to be OK at runtime in the docker images we build for production?  I wonder if it would be less risky to just copy the code over here, it's only 3 lines worth.

Reply:
Yeah! we have the technology!